collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -164,14 +171,17 @@ public void onCreate() {
      */
     private void setupStrictMode() {
         if (BuildConfig.DEBUG) {
-            StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder()
+            StrictMode.ThreadPolicy.Builder policyBuilder = new StrictMode.ThreadPolicy.Builder()
                     .detectAll()
-                    .permitDiskReads()  // shared preferences are being read on main thread
-                    .penaltyLog()
-                    .build());
-            StrictMode.setVmPolicy(new StrictMode.VmPolicy.Builder()
```

Comment:
I'm not sure what the merit in having logs for VM Policy was here - given the amount that come out of our app (and from Android itself), it's unlikely we'd spot anything.